sarif-2.2/prose/share/sarif-v2.2-draft.html

```diff
@@ -17869,10 +17867,10 @@ <h1 id="appendix-l-informative-revision-history-">
               <td style="text-align: left;">
                 Incorporate changes from GitHub issue <a href="https://github.com/oasis-tcs/sarif-spec/issues/375">#375</a>, <a href="https://github.com/oasis-tcs/sarif-spec/issues/376">#376</a> (tail), <a href="https://github.com/oasis-tcs/sarif-spec/issues/378">#378</a>, <a href="https://github.com/oasis-tcs/sarif-spec/issues/380">#380</a>,
                 <a href="https://github.com/oasis-tcs/sarif-spec/issues/381">#381</a>, <a href="https://github.com/oasis-tcs/sarif-spec/issues/382">#382</a>, <a href="https://github.com/oasis-tcs/sarif-spec/issues/383">#383</a>, <a href="https://github.com/oasis-tcs/sarif-spec/issues/387">#387</a>, <a href=
-                "https://github.com/oasis-tcs/sarif-spec/issues/389">#389</a>, <a href="https://github.com/oasis-tcs/sarif-spec/issues/390">#<span>https://github.com/oasis-tcs/sarif-spec/issues/390</span></a>, <a href="https://github.com/oasis-tcs/sarif-spec/issues/391">#391</a>, <a href=
```

Comment:
The changes from this point onwards look fine, but do I understand correctly they are just HTML re-rendering not related to the spec change itself? Should we do these separately?

Reply:
Honesty, I am not sure either way. On one hand, it is nice to have the two (for now) rendered artifacts in the same PR (I like to separate the commits though). On the other hand, the isolated markdown "snippet" / source files are rendered nicely enough for the reviewers to approve based on those alone.

In this case I did not render those two shared artifacts in a one time ...

I have three further change sets (the issues with agreed designs) in the pipeline and expect to have the renditions then only reflect the changes in the source files (plus possible changes in the date and so on.

For now thank you for the feedback and I will await the meeting tomorrow where we can maybe agree on the best way to proceed.

Reply:
Sounds good! It's clear enough with them in different commits, and the diff is not that confusing. Just wanted to be sure.